data/trie/patriciaMerkleTrie_test.go

```diff
@@ -16,7 +16,6 @@ import (
 	"github.com/ElrondNetwork/elrond-go/hashing"
```

Comment:
There are test for `ResetOldHashes` and `SetCheckpoint`: 
- TestTrieResetOldHashes
- TestTrieCheckpoint
- TestTrieCheckpointWithNoSnapshotCreatesSnapshot
- TestTrieSnapshottingAndCheckpointConcurrently

I chose to put them in different test files, because this is the trie_test package, and I don't have access to the unexported variables. If I were to create an export_test file, there would be a lot of functions and members that would have to be exported. There are some test that check that the trie operations work concurrently, but I will add more.